Dockerfile

```diff
@@ -1,46 +1,71 @@
 FROM balenalib/rpi-python:build AS builder
 
-ARG TARGET
-
-WORKDIR /usr/src/app
-
-
+WORKDIR /usr/src/
 
 RUN install_packages cmake libraspberrypi-dev git rsync binutils
 
-RUN pip install git+https://github.com/larsks/dockerize
+RUN git clone https://github.com/tasanakorn/rpi-fbcp.git
 
-RUN git clone https://github.com/juj/fbcp-ili9341.git
+WORKDIR /usr/src/rpi-fbcp
 
-WORKDIR /usr/src/app/fbcp-ili9341
+RUN sed '/target_link_libraries/d' -i CMakeLists.txt
+RUN echo "target_link_libraries(fbcp pthread bcm_host vchiq_arm vcos atomic)" >> CMakeLists.txt
 
-RUN sed -i '212 s/^/\/\//' config.h
+WORKDIR /usr/src/rpi-fbcp/build
 
-RUN sed -i '$d' CMakeLists.txt
+RUN cmake .. && make && mv fbcp /usr/bin/fbcp-dtoverlay
 
-RUN echo "target_link_libraries(fbcp-ili9341 pthread bcm_host vchiq_arm vcos atomic)" >> CMakeLists.txt
+RUN ldd /usr/bin/fbcp-dtoverlay
```

Comment:
hey @klutchell I'm not sure if we need this line?

other than that LGTM

Reply:
Oops! I'll remove it!